src/streams/components/rtsp/session.ts

```diff
@@ -113,6 +114,9 @@ export class RtspSession {
             }
             case 'rtcp': {
               this.recordNtpInfo(message)
+              if (isRtcpBye(message.rtcp)) {
+                this.clearKeepalive()
```

Comment:
I would rather that keepalive can fix itself, there are other ways it could potentially stop the instance from being garbage collected. There should be a `close` method on the class that cleans up these kind of resources, and the user should be forced to call it.